policy/release/rpm_repos/rpm_repos.rego

```diff
@@ -181,10 +185,18 @@ _is_rpmish(purl) if {
 	startswith(purl, "pkg:rpmmod/")
 }
 
-_known_repo_ids := lib.rule_data(_rule_data_key)
+_known_repo_ids := combined if {
+	extra := lib.rule_data(_rule_data_extras_key)
+	known := lib.rule_data(_rule_data_key)
+	combined := array.concat(extra, known)
+} else := known if {
+	known := lib.rule_data(_rule_data_key)
+}
```

Comment:
IIRC `lib.rule_data` returns `[]` when the key doesn't exist, so you could skip the else clause here if you wanted to.

Reply:
Ah! But if either is not an array (for whatever reason), then `_known_repo_ids` will produce no result having unintended consequences.